### PR TITLE
updated privacy notice

### PR DESCRIPTION
**Pull request by ssundahlTTD**
Updated privacy notice with modifications given by legal.

### CODE REVIEW

File: views_euid/privacy.hbs [hunk 1]
```diff
@@ -6,137 +6,159 @@
...
+      <li>If deemed necessary under applicable data protection legislation, the standard contractual clauses for data transfers, as approved by the European Commission Implementing Decision (EU) 2021/914 of 4 June 2021 for the transfer of Personal Data to third countries.</li>
```
Comment (ashleysmithTTD) [inline views_euid/privacy.hbs:81]: Should the data privacy link above be in href with a link and "here" like the other links in this PR?

File: views_euid/privacy.hbs [hunk 2]
```diff
@@ -6,137 +6,159 @@
...
+    <p>We may store your hashed email address or phone number for up to 18 months from the date on which the EUID was created in order to operate the EUID for certain advertisers and/or data partners. If you withdraw your consent, or object to our processing of your data for EUID, we keep a record of your opt-out, a hashed copy of the email address or phone number used to create the EUID, and the EUID itself. This is necessary for us to implement your request and ensure ongoing compliance with your data protection rights.</p>
+    
     <h2>Contact us</h2>
     <p>You may contact us regarding privacy matters as follows:</p>
-    <p>Email:<b>  privacy  </b>((at))<b>  thetradedesk  </b>((dot))<b>  com</b></p>
-    <p><strong>Global Data Protection Officer (DPO)</strong> The Trade Desk, Global Privacy Office 42 N. Chestnut St., Ventura, CA 93001 Email: <b>dpo</b> ((at)) <b>thetradedesk</b> ((dot)) <b>com</b></p>
+    <p>Email:<b>  privacy  </b>((at))<b>  thetradedesk  </b>((dot))<b>  com</b> or submit an inquiry <a href="https://privacyportal-eu.onetrust.com/webform/2abc1a63-35c5-4ef7-b11b-1c55714738b5/4c5ba598-b05c-4b1c-b3da-abab6ac1c9db">here</a></p>
```
Comment (ashleysmithTTD) [inline views_euid/privacy.hbs:91]: Is there a reason this isn't just written as privacy@thetradedesk.com ? And/or linked?

File: views_euid/privacy.hbs [hunk 3]
```diff
@@ -6,137 +6,159 @@
...
       <li>In the UK, we recommend you contact the Information Commissioner’s Office (ICO) of the United Kingdom.</li>
     </ul>
-    <h2>Changes to this policy</h2>
-    <p>We may revise this policy at any time. However, if we make material changes, we will not apply them retroactively.</p>
+
+    <h2>Changes to this notice</h2>
+    <p>We may revise this notice at any time. However, if we make material changes, we will not apply them retroactively.</p>
+
+    <h2>Last updated</h2>
+    <p>27 September 2024</p>
```
Comment (ashleysmithTTD) [inline views_euid/privacy.hbs:104]: This needs changing to today's date?

File: views_euid/privacy.hbs (hunk 1 above)
Comment (ssundahlTTD) [reply views_euid/privacy.hbs:81]: I just did what I was told:

image

File: views_euid/privacy.hbs (hunk 2 above)
Comment (ssundahlTTD) [reply views_euid/privacy.hbs:91]: Just doing what I'm told.  This probably keeps the spam away from web scrapes looking for emails.

image

File: views_euid/privacy.hbs (hunk 3 above)
Comment (ssundahlTTD) [reply views_euid/privacy.hbs:104]: I assume this date represents when the document was last updated by our legal team and not when it was updated on the site itself.  So I think this is the correct date.

**Pull request merged**